website/templates/share_registration.mako

```diff
@@ -0,0 +1,17 @@
+<%inherit file="base.mako"/>
+<%def name="title()">SHARE</%def>
+<%def name="content()">
+    <div id="share_registration_iframe"></div>
+</%def>
+
+<%def name="javascript_bottom()">
+    <script>
+        // Mako variables accessible globally
+        window.contextVars = $.extend(true, {}, window.contextVars, {
+            share: {
+                registration_url: '${registration_url | js_str}'
```

Comment:
Could improve namespacing like so:

```js
share: {
    urls: {
        register: '${registration_url | js_str}'
    }
}
```

This will make it cleaner if we add more URLS in the future.